packages/atomic/cypress/integration/search-box.cypress.ts

```diff
@@ -4,21 +4,15 @@ import {
   SearchBoxSelectors,
   generateAliasForSearchBox,
 } from '../selectors/search-box-selectors';
-import {
-  ResultListSelectors,
-  generateAliasForResultList,
-} from '../selectors/result-list-selectors';
 
 const queryText = 'test';
-const htmlCode =
-  '<atomic-search-box></atomic-search-box><atomic-result-list></atomic-result-list>';
+const htmlCode = '<atomic-search-box></atomic-search-box>';
 
 describe('SearchBox Test Suites', () => {
```

Comment:
Just a few notes, Search box should be in 2 words, also repeating the word in each test isn't necessary